### PR TITLE
Add Google Analytics

### PR DESCRIPTION
**Pull request by dh4u**
https://app.asana.com/0/1192620845659667/1199595507254378/f

I did have one question about the packages that suggested a npm audit fix that I put in the Asana card.

image

### CODE REVIEW

File: components/utils/analytics.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import ReactGA from 'react-ga';
+
+export const initGA = () => {
+  ReactGA.initialize('UA-113130296-3');
```
Comment (atlisg) [inline components/utils/analytics.js]: This is the FOOD-E ID, the correct ID is `UA-113130296-2`

**Review by atlisg — CHANGES_REQUESTED** [review]
Just the ID and we're good to go.
Note: Because this is a slightly different setup for the Layout component, I'm not sure if the pages will be tracked individually but let's just see what happens. If this only logs the main page, we can fix that quickly

Let's do the npm audit in a separate PR. Feel free to go for it if you like. :)

File: components/utils/analytics.js (hunk 1 above)
Comment (dh4u) [reply components/utils/analytics.js]: Updated.... https://github.com/ethicodeorg/website/commit/dcf8cea221b6212ac44c0bbf02380925d44374d2

**Review by atlisg — APPROVED** [review]
Looks good, feel free to merge. 👍

**Pull request merged**